src/test_l1_dsHdmiIn.c

```diff
@@ -344,7 +341,6 @@ void test_l1_dsHdmiIn_negative_dsHdmiInGetNumberOfInputs(void) {
     UT_LOG("\n Out %s\n", __FUNCTION__); 
 }
 
-
 /**
  * @brief Ensure dsHdmiInGetStatus() correctly fetches the HDMI input port status during positive scenarios.
  * 
```

Comment:
in the dsHdmiInSelectPort() we need to loop through it using dsHdmiInPort_t with all combinations dsVideoPlaneType_t

Reply:
Done